<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <resources><include location="../../../res/extui/extui.qrc"/></resources><author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>QG_CoordinateWidget</class>
  <widget class="QWidget" name="QG_CoordinateWidget">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>27</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy>
        <hsizetype>1</hsizetype>
        <vsizetype>1</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="minimumSize">
      <size>
        <width>380</width>
        <height>27</height>
      </size>
    </property>
    <property name="maximumSize">
      <size>
        <width>500</width>
        <height>50</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Coordinates</string>
    </property>
    <layout class="QHBoxLayout">
      <property name="margin">
        <number>0</number>
      </property>
      <property name="spacing">
        <number>0</number>
      </property>
      <item>
        <layout class="QVBoxLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>0</number>
          </property>
          <item>
            <widget class="QLabel" name="lCoord1">
              <property name="font">
                <font>
                  <family>Helvetica</family>
                  <pointsize>7</pointsize>
                </font>
              </property>
              <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
                <enum>QFrame::Plain</enum>
              </property>
              <property name="text">
                <string>Coordinates</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLabel" name="lCoord1b">
              <property name="font">
                <font>
                  <family>Helvetica</family>
                  <pointsize>7</pointsize>
                </font>
              </property>
              <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
                <enum>QFrame::Plain</enum>
              </property>
              <property name="text">
                <string>Coordinates</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <widget class="Line" name="line1">
          <property name="frameShape">
            <enum>QFrame::VLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QVBoxLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>0</number>
          </property>
          <item>
            <widget class="QLabel" name="lCoord2">
              <property name="font">
                <font>
                  <family>Helvetica</family>
                  <pointsize>7</pointsize>
                </font>
              </property>
              <property name="text">
                <string>Coordinates</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLabel" name="lCoord2b">
              <property name="font">
                <font>
                  <family>Helvetica</family>
                  <pointsize>7</pointsize>
                </font>
              </property>
              <property name="text">
                <string>Coordinates</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
</ui>
